gulpfile.js

```diff
@@ -25,9 +24,9 @@ if (!fs.existsSync(workDir)) {
     fs.mkdirSync(workDir);
 }
 
-gulp.task('build', ['render-datasource-templates', 'list-ga-services']);
-gulp.task('release', ['render-datasource-templates',/*'make-editor-schema', 'validate'*/]);
-gulp.task('watch', ['watch-datasource-templates']);
+gulp.task('build', ['update-lga-filter', 'render-datasource-templates', 'list-ga-services']);
```

Comment:
I don't think this will work.  `gulp` (in its infinite wisdom) executes all the tasks in the array in parallel.  So `render-datasource-templates` will happen before `update-lga-filter` finishes.